eng/pipelines/apiview-review-gen-typespec.yml

```diff
@@ -0,0 +1,70 @@
+pr: none
+
+trigger: none
+
+parameters:
+  - name: Reviews
+    type: string
+    default: '[{"ReviewID":"<reviewid>","RevisionID":"<revisionId>","SourceRepoName":"<RepoName>","FileName":"<fileName>","SourceBranchName":"<SourceBranchName>"}]'
+  - name: APIViewURL
+    type: string
+    default: 'https://apiview.dev' 
+
+pool:
+  name: azsdk-pool-mms-ubuntu-2204-general
+  vmImage: MMSUbuntu22.04
+
+variables:
+  NodeVersion: '18.x'
+
+jobs:
+- job: CreateSwaggerReviewCodeFile
```

Comment:
is Job something referenced somewhere else? or we can change the name? asking as it contains `Swagger` on it

Reply:
Thanks. I will update it.